app/uk/gov/hmrc/customs/notification/controllers/HeaderValidator.scala

```diff
@@ -17,7 +17,7 @@
 package uk.gov.hmrc.customs.notification.controllers
 
 import play.api.http.HeaderNames._
-import play.api.mvc.{ActionBuilder, AnyContent, BodyParser, ControllerComponents, Headers, Request, Result}
+import play.api.mvc._
```

Comment:
Could you revert this import change and the others in this PR? It just introduces noise in the commit. Also, explicit imports are better than wildcard.